site/zenodo_rdm/custom_fields.py

```diff
@@ -0,0 +1,154 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Custom fields."""
+
+from invenio_records_resources.services.custom_fields import (
+    BaseCF,
+    DoubleCF,
+    ISODateStringCF,
+    KeywordCF,
+    TextCF,
+)
+from invenio_rdm_records.contrib.imprint import (
+    IMPRINT_NAMESPACE,
```

Comment:
This is for rdm-records, but wouldn't it make more sense to remove prefixes of the actual custom field? (i.e. `NAMESPACE, CUSTOM_FIELDS`, etc.). Why:

- Shortens the names
- It does not add value to the name, since we are importing from the actual package, i.e. already in the `.contrib.imprint...`
- If there are collisions they can always be imported with `as ...`